fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/matching/structure/ValueMatchers.java

```diff
@@ -150,4 +152,16 @@ public static BindingMatcher<StreamableAggregateValue> streamableAggregateValue(
                 Extractor.of(StreamableAggregateValue::getChildren, name -> "children(" + name + ")"),
                 downstreamValues);
     }
+
+    @Nonnull
+    public static BindingMatcher<RecordTypeValue> recordTypeValue() {
+        return typed(RecordTypeValue.class);
+    }
+
+    @Nonnull
+    public static BindingMatcher<RecordTypeValue> recordTypeValue(@Nonnull BindingMatcher<CorrelationIdentifier> alias) {
```

Comment:
Same here, you get into stability problems if this is called outside the test cases (which have a debugger installed). If that's ok, then please go ahead (maybe add a comment into the java doc).